CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [Unreleased]
 
+### Changed
+The type restriction for file attachments has been removed.
```

Comment:
I think this should be a `Removed` entry, as per the Keep A Changelog advice. 🤔 

I'd also, be a bit clearer, something like...

```suggestion
### Removed
* The adaptor no longer checks incoming attachment content types aginst the `SUPPORTED_FILE_TYPES` list.
   This functionality can be implemented outside of the adaptor if still desired.
```